src/main/java/org/truffleruby/core/rope/RopeGuards.java

```diff
@@ -21,4 +24,16 @@ public static boolean isLeafRope(Rope rope) {
         return rope instanceof LeafRope;
     }
 
+    public static boolean isEmpty(byte[] bytes) {
+        return bytes.length == 0;
+    }
+
+    public static boolean isBinaryString(Encoding encoding) {
```

Comment:
Binary isn't exactly the same thing as ASCII is it?

Reply:
Binary == ASCII-8BIT, which ends up being ASCIIEncoding.